src/packages/media/media/dropzone/dropzone.element.ts

```diff
@@ -26,14 +26,14 @@ export class UmbDropzoneElement extends UmbLitElement {
 		return this._disableFolderUpload;
 	}
 	public set disableFolderUpload(isAllowed: boolean) {
-		this.dropzoneManager.setIsFoldersAllowed(!isAllowed);
+		this.#dropzoneManager.setIsFoldersAllowed(!isAllowed);
 	}
 	private readonly _disableFolderUpload = false;
 
 	@state()
 	private _progressItems: Array<UmbUploadableItem> = [];
 
-	public dropzoneManager: UmbDropzoneManager;
+	#dropzoneManager: UmbDropzoneManager;
```

Comment:
While this change is nice, it is unfortunately a breaking change.

Reply:
@iOvergaard i dont think this part is in release version yet if I'm not mistaken, it was merged in just the other day.

Reply:
Oh great, then disregard my comment =)